Description: patch Rakefile to allow for easier manpage generation
 This patch disable some of the dependencies that are not
 required to build the actual manpage.
Author: Simon Elsbrock
Forwarded: no
---
Index: jq/docs/Rakefile
===================================================================
--- jq.orig/docs/Rakefile	2013-05-19 10:48:18.000000000 +0000
+++ jq/docs/Rakefile	2014-02-18 08:04:58.000000000 +0000
@@ -1,11 +1,6 @@
 require 'yaml'
-require 'bonsai'
-require 'liquid'
-require 'maruku'
-require 'json'
 require 'ronn'
 require 'tempfile'
-require 'yaml'
 
 module ExtraFilters
   def markdownify(input)
@@ -26,7 +21,7 @@
   end
 end
 
-Liquid::Template.register_filter(ExtraFilters)
+#Liquid::Template.register_filter(ExtraFilters)
 
 
 task :serve do
